src/Core_Fix_ResourceUnloadOptimizations/Core.ResourceUnloadOptimizations.cs

```diff
@@ -71,6 +75,12 @@ private IEnumerator CleanupCo()
                     if (--_garbageCollect == 0)
                         RunGarbageCollect();
                 }
+
+                if( _forceCleanup > 0 )
+                {
+                    if (--_forceCleanup == 0)
+                        RunForceCleanupIfNecessary();
+                }
```

Comment:
This seems to never run because _forceCleanup starts at 0 and is never assigned.

Reply:
Fixed. Thanks for pointing this out.  




> didn't get commited?  

No. Forgot to revert HSUS!